packages/config/src/Config.ts

```diff
@@ -325,7 +325,11 @@ export async function modifyConfigAsync(
   modifications: Partial<ExpoConfig>,
   readOptions: GetConfigOptions = {},
   writeOptions: WriteConfigOptions = {}
-): Promise<{ type: 'success' | 'warn' | 'fail'; message?: string; config: ExpoConfig | null }> {
+): Promise<{
+  type: 'success' | 'warn' | 'fail';
+  message?: string;
+  config: ExpoConfig | AppJSONConfig | null;
+}> {
```

Comment:
I'm not a huge fan of defining complex types inline. I'd prefer to define it separately.

Reply:
Thanks for the suggestion, but I'd rather leave it like this. If it was used in many places or it would somehow be important that it has a name, I'd consider pulling it out. But here, it's more important to have this information close to the function.